docs/get-started/quickstarts/control-motor.md

```diff
@@ -21,30 +21,41 @@ Follow these steps to control your motor:
 
 {{< expand "Step 1: Create a machine" >}}
 
-Go to the Viam app and [add a new machine](/cloud/machines/#add-a-new-machine).
-
-![The 'First Location' page on the Viam app with a new machine name in the New machine field and the Add machine button next to the field highlighted.](/fleet/app-usage/create-machine.png)
+Go to the Viam app.
+Select a location and [add a new machine](/cloud/machines/#add-a-new-machine).
 
 {{< /expand >}}
+{{%expand "Step 2: Install viam-server" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+
+{{% /expand%}}
```

Comment:
Just merged a PR yesterday changing a bunch of these--it seems that when we use % sometimes weird formatting things occur. This may be the reason that in the staging of this PR right now, there is an extra gap between this expander and the next one.
```suggestion
{{< expand "Step 2: Install viam-server" >}}

Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.

{{< /expand >}}
```